0.17/files/map-gen-settings-template.json

```diff
@@ -0,0 +1,70 @@
+{
+	"terrain_segmentation": templateGenTerrainSegmentation,
+	"water": templateGenTerrainWater,
+	"width": templateGenMapWidth,
+	"height": templateGenMapHeight,
+	"starting_area": templateGenMapStartingArea,
+	"peaceful_mode": templateGenPeacefullMode,
+	"autoplace_controls": {
+		"coal": {
+			"frequency": templateGenCoalFrequency,
+			"size": templateGenCoalSize,
+			"richness": templateGenCoalRichness
+		},
+		"stone": {
+			"frequency": templateGenStoneFrequency,
+			"size": templateGenStoneSize,
+			"richness": templateGenStoneRichness
+		},
+		"copper-ore": {
+			"frequency": templateGenCopperFrequency,
+			"size": templateGenCopperSize,
+			"richness": templateGenCopperRichness
+		},
+		"iron-ore": {
+			"frequency": templateGenIronFrequency,
+			"size": templateGenIronSize,
+			"richness": templateGenIronRichness
+		},
+		"uranium-ore": {
+			"frequency": templateGenUraniumFrequency,
+			"size": templateGenUraniumSize,
+			"richness": templateGenUraniumRichness
+		},
+		"crude-oil": {
+			"frequency": templateGenCrudeFrequency,
+			"size": templateGenCrudeSize,
+			"richness": templateGenCrudeRichness
+		},
+		"trees": {
+			"frequency": templateGenTreesFrequency,
+			"size": templateGenTreesSize,
+			"richness": templateGenTreesRichness
+		},
+		"enemy-base": {
+			"frequency": templateGenBiterFrequency,
+			"size": templateGenBiterSize,
+			"richness": templateGenBiterRichness
+		}
+	},
+	"cliff_settings": {
+		"name": "templateGenCliffName",
+		"cliff_elevation_0": templateGenCliffElevationZero,
+		"cliff_elevation_interval": templateGenCliffElevationInterval,
+		"richness": templateGenCliffRichness
+	},
+	"property_expression_names": {
+		"elevation": "templateGenExpressionElevation",
```

Comment:
Did you miss some ``"`` here?

Reply:
those are in quotes in the example. so I left them in quotes. It was a bit of a strange section in the file to be honest.